composer.json

```diff
@@ -6,13 +6,13 @@
         }
     },
     "require": {
-        "laravel/framework": "~5.1"
+        "laravel/framework": "~6.0"
```

Comment:
Use laravel 6 framework, removing 5.1 as some of the other required libraries are compatible either for laravel 5 or for laravel 6 but not both